src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-sourceFolder.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    <p>Optionally specify the location of the folder containing source code, relative to the project root folder. The specified folder and its subfolders are added to the top of the MATLAB search path. To generate a code coverage report, MATLAB uses only the source code in the specified folder and its subfolders. </p>
```

Comment:
This looks good. I would remove "Optionally". I would also use "folders" instead of folder because we can have multiple folders.
Is source folder useful for Simulink Test model coverage report as well? If so, please drop the word "code". So, the text would become:

Specify the location of folders containing source code, relative to the project root folder. The specified folders and their subfolders are added to the top of the MATLAB search path. To generate a coverage report, MATLAB uses only the source code in the specified folders and their subfolders.

Reply:
Thanks for review. I changed the text as per your suggestion.